mayhem/xhtml2pdf_fuzz.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
```

Comment:
It looks like you're only raising exceptions after the first 1000 iterations. Can you elaborate here as well? If there is an error that occurs too often, it may be worth trying to craft more specific fuzzed data, or ensuring that the  api contract is being respected.

Reply:
I've made some update and removed the skipping of first 1000 iterations. I was skipping to make sure nothing was crashing too soon before fuzz actually occurred, it seems that if an exception is thrown too early it won't give better feedback to me on mayhem. Right now I'm trying to get the test cases faster but doesn't seem possible with that function call.